next-app/.npmrc

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
nice!  We should also put this in the root folder to prevent the same sort of issues with wrong node version

Reply:
yeah, that's probably a good idea, I like the idea of doing that when we move the Makefile to the root of the project, are you ok with giving yourself a reminder for this one?

Reply:
yup, I'm looking forward to the challenge :)